pkgs/misc/rivalcfg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, python3Packages }:
+
+with python3Packages; buildPythonPackage rec {
```

Comment:
I've never seen `with python3Packages;` used before, do you have an example where this has been used somewhere

Reply:
Yes, in tools/text/invoice2data. I grepped for buildPythonPackage since I was getting an error and copied from that package. Should I be using python37Packages?

Reply:
I am not sure, I am not a python expert. My query was more about using with instead of `python3Packages.buildPythonPackage` here and moving the `with python3Packages` to the buildInputs. It can be confusing to have big with clauses, for example it is not obvious that the build inputs are coming form `python3Packages`

Reply:
python3Package, python37Packages, python3.pkgs are all aliases of each other.